content/docs/dvclive/api-reference/live/index.md

```diff
@@ -38,6 +38,9 @@ other metadata.
 - `dir` - Location of the directory to store
   [outputs](/doc/dvclive/get-started#outputs).
 
+- `params_path` - `{Live.params_path}`. Location of the parameters logged via
+  [`log_param`](/doc/dvclive/api-reference/live/log_param#description)/[`log_params`](/doc/dvclive/api-reference/live/log_params#description).
+
 - `summary_path` - `{Live.dir}.json`. Location of the
```

Comment:
cc @daavoo Should this be called a "metrics summary" now to distinguish from params? Should we update the attribute name?

Reply:
+1

Maybe we should also consider rethinking the summary/report names, they're somehow similar and it can be confusing